src/main/java/dao/ShopDaoImpl.java

```diff
@@ -0,0 +1,46 @@
+package dao;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class ShopDaoImpl implements ShopDao {
+    private static final String FIRST_LINE_REPORT = "fruit,quantity";
+
+    @Override
+    public List<String> readFromFile(File file) {
+        List<String> result = new ArrayList<>();
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
+            bufferedReader.readLine();
+            String value = bufferedReader.readLine();
+            if (value == null) {
+                return Collections.emptyList();
+            }
+            while (value != null) {
+                result.add(value);
+                value = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read the file" + file, e);
+        }
+        return result;
```

Comment:
```suggestion
        try {
            return Files.readAllLines(Path.of(file.getPath()));
        } catch (IOException e) {
            throw new RuntimeException("Can't read the file" + file, e);
        }
```
Better use this method instead of writing logic from scratch